google-services-plugin/src/main/groovy/com/google/gms/googleservices/GoogleServicesPlugin.groovy

```diff
@@ -66,12 +40,15 @@ class GoogleServicesPlugin implements Plugin<Project> {
       if (config.disableVersionCheck) {
         return
       }
-      DependencyAnalyzer globalDependencies = new DependencyAnalyzer()
+
```

Comment:
Any reason why this is only commented out and not deleted?

Reply:
This was not meant to be commented out, was only done for testing. Fixed